PVPCPlanner/View/Components/CustomLazyList.swift

```diff
@@ -0,0 +1,70 @@
+import SwiftUI
+
+enum CustomLazyListDirection {
+    case vertical
+    case horizontal
+}
+
+struct CustomLazyList<Content: View>: View {
+    var spacing: CGFloat
+    var listDirection: CustomLazyListDirection
+    var backgroundColor: Color
+    var content: () -> Content
+
+    init(spacing: CGFloat = 10,
+         listDirection: CustomLazyListDirection = CustomLazyListDirection.vertical,
+         backgroundColor: Color = Color.white,
+         @ViewBuilder content: @escaping () -> Content) {
+
+        self.spacing = spacing
+        self.listDirection = listDirection
+        self.backgroundColor = backgroundColor
+        self.content = content
+    }
```

Comment:
Este init esta linteado?

Reply:
Con el init y el lintado tengo un problema y es que si hago lo de command + S se cambia el { de linea , pero entonces aparece un warning

"Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration (opening_brace)"

![imagen](https://github.com/user-attachments/assets/bb53a1f9-03b9-4060-ae3e-945c69a17ab6)

Por eso lo dejé así

Por el resto no cambia nada